packages/engine/src/scene/functions/loaders/ColliderFunctions.test.ts

```diff
@@ -1,72 +1,116 @@
 import assert from 'assert'
-import { Mesh, MeshNormalMaterial, Quaternion, SphereBufferGeometry, Vector3 } from 'three'
+import proxyquire from 'proxyquire'
+import { Object3D, Quaternion, Vector3 } from 'three'
 
 import { ComponentJson } from '@xrengine/common/src/interfaces/SceneInterface'
 
 import { Engine } from '../../../ecs/classes/Engine'
-import { createWorld } from '../../../ecs/classes/World'
+import { Entity } from '../../../ecs/classes/Entity'
+import { createWorld, World } from '../../../ecs/classes/World'
 import { addComponent, getComponent, hasComponent } from '../../../ecs/functions/ComponentFunctions'
 import { createEntity } from '../../../ecs/functions/EntityFunctions'
-import { getGeometryScale } from '../../../physics/classes/Physics'
 import { ColliderComponent } from '../../../physics/components/ColliderComponent'
 import { CollisionComponent } from '../../../physics/components/CollisionComponent'
-import { BodyType } from '../../../physics/types/PhysicsTypes'
-import { TransformComponent } from '../../../transform/components/TransformComponent'
+import { EntityNodeComponent } from '../../components/EntityNodeComponent'
 import { Object3DComponent } from '../../components/Object3DComponent'
-import { deserializeCollider } from './ColliderFunctions'
+import { SCENE_COMPONENT_COLLIDER } from './ColliderFunctions'
+
+globalThis.PhysX = {
+  PxBoxGeometry: class {}
```

Comment:
we should just initialise physx as we do in other tests, less chance of this interferring with other tests